.travis.yml

```diff
@@ -0,0 +1,34 @@
+addons:
+  apt:
+    packages:
+    - libgmp-dev
+before_install:
+- sh script/install-stack.sh
+cache:
+  directories:
+  - "${HOME}/.local/bin"
+  - "${HOME}/.stack"
+  - ".stack-work"
+env:
+  global:
+  - CASHER_TIME_OUT=1000
+  - BUILD="elmi-to-json-${TRAVIS_OS_NAME}.tar.gz"
+language: c
+os:
+- linux
+- osx
+sudo: false
+script:
+- stack setup
+- stack test
+- stack build --ghc-options -O2
+- tar zcvf "${BUILD}" -C "$(stack path --local-install-root)/bin" elmi-to-json
```

Comment:
I think it's nice to include the version number in the filename of the tgz file  (like `elmi-to-json-1.0.0-linux.tgz`) to prevent confusion for people who download them.

Reply:
sgtm.